examples/linear_model/plot_lasso_coordinate_descent_path.py

```diff
@@ -20,6 +18,8 @@
 from sklearn.linear_model import lasso_path, enet_path
 from sklearn import datasets
 
+print(__doc__)
+
```

Comment:
If you leave this print on the top of the file, after the docstring, I think that sphinx-gallery hides it, so I think it's better (should be checked, though)